README.md

```diff
@@ -1,81 +1,41 @@
 # INI parser library
 
 This is a very simple ini-parser library that provides:
+
 - Raw record reading
 - Leading/trailing whitespace removal
 - comments based on `;` and `#`
 - Zig API
 - C API
 
-## Usage example
-
-### Zig 
+## Example
 
-```zig
-const std = @import("std");
-const ini = @import("ini");
+Please see the flolder `example`!
```

Comment:
Please keep the code examples in the `README.md`, but also link to `example` like this:
```md
Examples can be found in the folder [`example`](./example).
```